index.html

```diff
@@ -2349,6 +2349,94 @@ <h2>Hypermedia Controls Vocabulary Definitions</h2>
           </table>
         </div>
 
+        <p>
+          A <a>Thing Description</a> of a WoT producer may have multiple forms entries with, e.g., different 
+          protocol and/or content types declarations that a <a>Consumer</a> could possibly support. In that case 
+          the Consumer may choose any form entry that works (e.g., the protocol and content type is supported) for them. 
+          When one form is chosen, it is expected that the <a>Consumer</a> will continue to use it as long as possible 
+          for every new interaction with the WoT producer.
```

Comment:
What do we mean by that? Under which circumstances is it no longer posible to use a form? What should happen in this case?

Reply:
I think this was a discussion somewhere else as well. In general, we do not specify consumer behavior in the TD spec.

Reply:
So it might be a good point to remove this or have a separate section and reference to that